docs/integrations/glue_hive_metastore.md

```diff
@@ -1,71 +1,68 @@
 ---
 layout: default
-title: Glue / Hive metastore 
-description: When working with Glue or Hive metastore, each table can point only to one lakeFS branch. 
+title: Glue / Hive metastore
+description: When working with Glue or Hive metastore, each table can point only to one lakeFS branch.
 parent: Integrations
 nav_order: 65
 has_children: false
 redirect_from: ../using/glue_hive_metastore.html
 ---
 
-# Glue / Hive metastore 
-{: .no_toc }
-When working with Glue or Hive metastore, each table can point only to one lakeFS branch.
-When creating a new branch, you may want to have a similar table pointing to the new branch.  
-To support this, lakeFS comes with cli commands which enable copying metadata between branches.
-
-
-## Table of contents
+# Table of contents
 {: .no_toc .text-delta }
 
 1. TOC
 {:toc}
+   
+# Glue / Hive Metastore Intro
+This part contains a brief explanation about how Glue/Hive metastore works with lakeFS 
+
+Glue/Hive Metastore stores metadata related to Hive and other services (such as Spark and Trino).
+The Metastore contains metadata such as the location of the table, information about columns, partitions and many more.
 
+## Without lakeFS 
+{: .no_toc }
+In order to query the table `my_table`, Spark will: 
+* Request the metadata from Hive Metastore (steps 1,2)
+* Use the location from the metadata to access the data in S3 (steps 3,4).
+![metastore with S3](../assets/img/metastore-S3.svg)
+  
+<br/><br/>
+
+## With lakeFS
+{: .no_toc }
+When using lakeFS, the flow stays exactly the same. Note that the location of the table `my_table` now contains the branch `s3://example/main/path/to/table`
+![metastore with S3](../assets/img/metastore-lakefs.svg)
 
 
-# Access to metastores
-The commands could run on Glue or Hive metastore, each of them have different parameters and different configurations.
-## Hive
-To run a command on Hive metastore we will need to:
+<br/><br/><br/>
 
-Specify that the type is Hive ``` --type hive```
+# Managing Tables With lakeFS Branches
+## Motivation
+When creating a table in Glue/Hive metastore (using a client such as Spark, Hive, Presto), We specify the table location.
+Consider the table `my_table` which was created with the location `s3://example/main/path/to/table`.
 
-Insert the Hive metastore uri in the metastore-uri flag ``` --metastore-uri thrift://hive-metastore:9083```
+Assume we created a new branch called `DEV` with `main` as the source branch. 
+The data from `s3://example/main/path/to/table` is now accessible in `s3://example/DEV/path/to/table`.
+The metadata is not managed in lakeFS, meaning we don't have any table pointing to `s3://example/DEV/path/to/table`.
 
-It's recommended to configure these fields in the lakectl configuration file:
+To address this lakeFS introduces `lakectl metastore` commands. The case above could be handled using the copy command: it can create a copy of `my_table` with data located in `s3://example/DEV/path/to/table`.
```

Comment:
```suggestion
To address this, lakeFS introduces `lakectl metastore` commands. The case above could be handled using the `copy` command: it can create a copy of `my_table` with data located in `s3://example/DEV/path/to/table`. Note that this is a fast, metadata-only operation.
```